Source/WebKit/webpushd/WebPushDaemon.mm

```diff
@@ -125,12 +125,12 @@ static bool platformShouldPlaySound(const WebCore::NotificationData& data)
 #endif
 }
 
-static NSString *platformDefaultActionBundleIdentifier(PushClientConnection& connection)
+static NSString *platformDefaultActionBundleIdentifier(const WebCore::PushSubscriptionSetIdentifier& identifier)
 {
 #if PLATFORM(IOS)
-    if (connection.hostAppCodeSigningIdentifier() == "com.apple.SafariViewService"_s)
+    if (identifier.bundleIdentifier == "com.apple.SafariViewService"_s)
         return @"com.apple.webapp";
-    return (NSString *)connection.hostAppCodeSigningIdentifier();
+    return (NSString *)identifier.bundleIdentifier;
 #else
     // FIXME: Calculate appropriate value on macOS
     return nil;
```

Comment:
If we want to enable declarative push for macOS Dock apps then this does not seem correct...

Reply:
For now this allows us to API test on Mac, which I think is the highest immediate priority.
This will be revisited in the near future when I get to that.